[INFO -- 2017-03-09 19:04:29]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-09 19:04:29]
Initialization strings:
[INFO -- 2017-03-09 19:04:29]
(1) target extended-remote | ssh -T mic0 gdbserver --multi -
[INFO -- 2017-03-09 19:04:29]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-09 19:04:29]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-09 19:04:29]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-09 19:04:29]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-09 19:04:29]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-03-09 19:04:29]
initSignal:6
[INFO -- 2017-03-09 19:04:29]
endSignal:8
[INFO -- 2017-03-09 19:04:29]
seqSignal:30
[INFO -- 2017-03-09 19:04:29]
faultMode:0
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[EXCEPTION -- 2017-03-09 19:04:47]
pythonException: invalid frame
[INFO -- 2017-03-09 19:04:47]
Backtrace BEGIN:
#0  0x00000000004033b5 in L_main_130__par_loop0_2_0 () at lud_check.c:134
#1  0x00007ffff790ba1f in ?? ()
#2  0x0000000000000000 in ?? ()

129	    int i, host_errors = 0;
130	#pragma omp parallel for reduction(+:host_errors)
131		for ( i = 0; i < matrix_dim; i++) {
132			int j;
133			for ( j = 0; j < matrix_dim; j++) {
134	                	if ((fabs((m[i + matrix_dim * j] - gold[i + matrix_dim * j]) / m[i + matrix_dim * j]) > 0.0000000001) || (fabs((m[i + matrix_dim * j] - gold[i + matrix_dim * j]) / gold[i + matrix_dim * j]) > 0.0000000001)) {
135				//if (m[i + matrix_dim * j] != gold[i + matrix_dim * j]) {
136					char error_detail[200];
137					sprintf(error_detail," p: [%d, %d], r: %1.16e, e: %1.16e", i, j, m[i + matrix_dim * j], gold[i + matrix_dim * j]);
138					host_errors++;

Backtrace END
Memory content before bitflip:0x000x000x000x00
Fault Mode: Single bit-flip
base address to flip value: 0x7fffffffe2f0
address max offset: 4
Memory content after  bitflip:0x000x000x000x20
frame name: L_main_130__par_loop0_2_0
symbol name: host_errors
symbol filename: lud_check.c
symbol line: 129
value: 536870912
value address: 0x7fffffffe2f0
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-09 19:04:47]
Fault Injection Successful after 18.7552471161s
[INFO -- 2017-03-09 19:04:48]
event type: exit
[INFO -- 2017-03-09 19:04:48]
exit code: 1
